framework/PostProcessorFunctions/HistorySetSnapShot.py

```diff
@@ -264,8 +261,6 @@ def historySetWindow(inputDic,timeStepID,inpVars,outVars,N,pivotParameter):
     @ Out, outDic, dict, it contains the temporal slice of all histories
   """
   outputDic={'data':{}}
-  outputDic['data']['ProbabilityWeight'] = inputDic['data']['ProbabilityWeight']
-  outputDic['data']['prefix'] = inputDic['data']['prefix']
```

Comment:
why the metadata is not added back here?

Reply:
they are added in the loop below:

```
  # collect metadata, if it exists, to pass through
  for key in inputDic['metaKeys']:
    outputDic['data'][key] = inputDic['data'][key]
```